packages/dds/tree/README.md

```diff
@@ -97,7 +110,14 @@ could be added in the future.
 
 #### Viewing
 
-TODO: add a diagram for this section.
+```mermaid
+graph TD;
```

Comment:
suggestion: A L->R graph might make this a bit easier to read. To me, top-down feels more like ownership rather than data flow (that might just be the pipe influence though).

Reply:
ya, that's much better. I didn't realize it was so easy to do that.